components/form/MultiSelect.js

```diff
@@ -0,0 +1,79 @@
+import { twMerge } from "tailwind-merge"
+import { useState, useEffect, useRef } from "react"
+import ArrowDown from "../icons/ArrowDown"
+
+export default function MultiSelect({
+    className = '',
+    onChange = () => { },
+    options = [],
+    selected = [],
```

Comment:
We can initially include the `Newsletter` list item as a default setting. Though the current approach is also nice 🚀

Reply:
The fact that I didn't add default value here because `MultiSelect` is a component. So, it should be made extensible and reusable in other parts of the website as well. Hence, we shouldn't add default values of `Newsletter` here.

Reply:
Can we extend the component to accept an optional prop to pass to the `selected` state? Can improve UX significantly.

Reply:
Can you pls explain this? 

`component to accept an optional prop to pass to the selected state`

What do you mean by this statement?

Reply:
I think you got confused over here. `selectedSubscriptions` means the subscriptions that is selected by the user in the dropdown. And Initially, it will be an empty array. I'm referencing this line - https://github.com/asyncapi/website/pull/2315/files#diff-c6e55866aad6f0d6916876be72833feed45f97662c3596a14c4ab3f38aa9f2daR18.

Reply:
Yeah, I am referencing the same line. My point is to take the newsletter item in that array initially (by hardcoding it).

Reply:
I don't prefer that approach because I have not made `MultiSelect.js` for Newsletter only. It's a reusable component, so it can be used at multiple places. Hence, hardcoding anything inside the component may give unusual errors, when used at different place.